test/unit/clarity_ext/helpers.py

```diff
@@ -244,18 +244,18 @@ def analytes_to_result_files_set():
 
 
 def print_out_dict(object_list, caption):
-    print("{}:".format(caption))
+    print(("{}:".format(caption)))
```

Comment:
`2to3` is confused here - the extra parentheses have no effect.

Not that it's really important, but I think I know why it thinks this is required. It sees the parentheses in Python2 and assumes we were trying to print **a tuple**.